main.js

```diff
@@ -143,6 +143,12 @@ function validateIcons() {
   }
 }
 
+function checkInputs(secondsValue, minutesValue, intentionsValue) {
+  (intentionsValue) ? hideError(1) : addError(1);
```

Comment:
Thank you for including a ternary operator here - it really cleaned up the previous function.

Reply:
Yes love the ternaries!